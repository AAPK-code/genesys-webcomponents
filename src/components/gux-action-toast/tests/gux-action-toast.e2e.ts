import { newE2EPage } from '@stencil/core/testing';

describe('gux-action-toast', () => {
  it('renders', async () => {
    const page = await newE2EPage();

    await page.setContent('<gux-action-toast></gux-action-toast>');
    const element = await page.find('gux-action-toast');
    expect(element).toHaveClass('hydrated');
  });
});
