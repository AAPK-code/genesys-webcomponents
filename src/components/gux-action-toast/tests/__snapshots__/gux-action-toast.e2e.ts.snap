// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-action-toast #render should render action toast 1`] = `"<gux-action-toast lang=\\"en\\" accent=\\"neutral\\" class=\\"hydrated\\"><!----> <div class=\\"header\\"><div class=\\"icon\\"><gux-icon slot=\\"icon\\" icon-name=\\"add-user\\" decorative=\\"\\" class=\\"hydrated\\"><svg aria-hidden=\\"\\" viewBox=\\"0 0 32 32\\"><use xlink:href=\\"#gux-icon-add-user\\"></use></svg></gux-icon></div><div class=\\"title\\"><div slot=\\"title\\">Title</div></div></div><div class=\\"message\\"><div slot=\\"message\\">This is the message</div></div><div class=\\"action-buttons\\"><div class=\\"negitive-button\\"><gux-button slot=\\"negitive-button\\" class=\\"hydrated\\"><!----><button class=\\"gux-secondary\\">Reject</button></gux-button></div><div class=\\"positive-button\\"><gux-button slot=\\"positive-button\\" accent=\\"primary\\" class=\\"hydrated\\"><!----><button class=\\"gux-primary\\">Accept</button></gux-button></div></div></gux-action-toast>"`;
